response_operations_ui/views/reporting_units.py

```diff
@@ -63,9 +59,9 @@ def view_reporting_unit(ru_ref):
             "title": f"{ru_ref}"
         }
     ]
-    return render_template('reporting-unit.html', ru_ref=ru_ref, party_id=respondent.get('id'),
```

Comment:
Still need to pass the party_id argument.

Reply:
We don't actually need this. Previously the verification email link was using this party_id but this is wrong because we need to get the id for the selected respondent. This just returns the last one